entity-framework/core/miscellaneous/platforms.md

```diff
@@ -14,25 +14,31 @@ We want EF Core to be available to developers on all modern .NET implementations
 
 The following table provides guidance for each .NET implementation:
 
-| EF Core                       | 3.1        | 5.0             |
-|:------------------------------|:-----------|:----------------|
-| .NET Standard                 | 2.0        | 2.1             |
-| .NET Core                     | 2.0        | 3.0             |
-| .NET Framework<sup>(1)</sup>  | 4.7.2      | (not supported) |
-| Mono                          | 5.4        | 6.4             |
-| Xamarin.iOS<sup>(2)</sup>     | 10.14      | 12.16           |
-| Xamarin.Mac<sup>(2)</sup>     | 3.8        | 5.16            |
-| Xamarin.Android<sup>(2)</sup> | 8.0        | 10.0            |
-| UWP<sup>(3)</sup>             | 10.0.16299 | TBD             |
-| Unity<sup>(4)</sup>           | 2018.1     | TBD             |
+EF Core                       | 3.1        | 5.0    | 6.0
+----------------------------- | ---------- | ------ | ---
+.NET                          |            | 5.0    | 6.0
```

Comment:
Maybe ".NET (previously .NET Core)" to avoid the confusion of two lines?

Reply:
I played with both ways and couldn't decide. I'll combine again.